packages/frontend-2/components/settings/workspaces/Billing.vue

```diff
@@ -38,14 +44,16 @@
                 {{ isTrialPeriod ? 'First payment due' : 'Next payment due' }}
```

Comment:
<img width="681" alt="image" src="https://github.com/user-attachments/assets/8dcb26b8-16d7-48b1-8148-71b4727b0f62">
Just checking that the pretitle `Current plan` should be font-medium, but the other pretitles are not.

Reply:
No, it wasn't supposed to be 😁